core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsTests.java

```diff
@@ -1015,6 +1015,14 @@ private void ensureStableCluster(int nodeCount, TimeValue timeValue, @Nullable S
                     + stateResponse.getState().prettyPrint());
         }
         assertThat(clusterHealthResponse.isTimedOut(), is(false));
+        if (validateOtherNodes) {
```

Comment:
yeah, I cherry picked this from another place and this came along.. I'll update the pr description.